ProcessList.c

```diff
@@ -300,9 +301,22 @@ void ProcessList_scan(ProcessList* this) {
 
    for (int i = Vector_size(this->processes) - 1; i >= 0; i--) {
       Process* p = (Process*) Vector_get(this->processes, i);
-      if (p->updated == false)
-         ProcessList_remove(this, p);
-      else
+      if (p->tomb_time > 0) {
+         // remove tombed process
+         if (time(NULL) >= p->tomb_time) {
+            ProcessList_remove(this, p);
+         }
+      } else if (p->updated == false) {
+         // process no longer exists
+         if (this->settings->highlightNewOldNsec > 0) {
+            // mark tombed
+            p->tomb_time = time(NULL) + this->settings->highlightNewOldNsec;
```

Comment:
This will fail in case of leap seconds and adjtime calls.
Is that corner case enough to care for something like htop (i.e. use difftime instead)?

Reply:
Prefer [https://linux.die.net/man/3/clock_gettime](`clock_gettime`9 with `CLOCK_MONOTONIC`. This avoids the leap second issue AND is intended for such cases.